addons/mail/static/src/js/followers.js

```diff
@@ -327,7 +327,7 @@ var Followers = AbstractField.extend({
                 var is_channel = $currentTarget.data('oe-model') === 'mail.channel';
                 self.dialog = new Dialog(this, {
                     size: 'medium',
-                    title: _t('Edit Subscription of ') + $currentTarget.siblings('a').text(),
+                    title: _t('Edit Default Notifications of ') + $currentTarget.siblings('a').text(),
```

Comment:
"Default Notifications" ? It is not correct. It has nothing to do with default. On a task I edit my subscription or my followed subtypes, but not default notifications.